GNUmakefile

```diff
@@ -55,7 +55,11 @@ WITH    ?= --without doc
 #WITH    ?= --without=doc --with=gcov
 
 LAST_RC		?= $(shell test -e /Volumes || git tag -l | grep Pacemaker | sort -Vr | grep rc | head -n 1)
-LAST_RELEASE	?= $(shell test -e /Volumes || git tag -l | grep Pacemaker | sort -Vr | grep -v rc | head -n 1)
+ifneq ($(origin VERSION), undefined)
+LAST_RELEASE	?= Pacemaker-$(VERSION)
+else
+LAST_RELEASE	?= $(shell git tag -l | grep Pacemaker | sort -Vr | grep -v rc | head -n 1)
+endif
```

Comment:
The /Volumes test is a workaround for OS X sort not having the -V sorting option